Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 1
	at PiPair.main(PiPair.java:32)
